graspy/align/seedless_procrustes.py

```diff
@@ -0,0 +1,353 @@
+    inside of the test for the equivalence of the latent distributions, and no
+    1-1 matching between the vertices of the two graphs can be established.
+
+    Parameters
+    ----------
+        optimal_transport_lambda : float (default=0.1), optional
+            Regularization term of the Sinkhorn optimal transport algorithm.
+
+        optimal_transport_eps : float (default=0.01), optional
+            Tolerance parameter for the each Sinkhorn optimal transport
+            algorithm. I.e. tolerance for each "E-step".
+
+        optimal_transport_num_reps : int (default=1000), optional
+            Number of repetitions in each iteration of the iterative optimal
+            transport problem. I.e. maximum number of repetitions in each
+            "E-step".
+
+        iterative_num_reps : int (default=100), optional
+            Number of reps in each iteration of the iterative optimal transport
+            problem. I.e. maxumum number of total iterations the whole "EM"
+            algorithm.
+
+        init : string, {'2d' (default), 'sign_flips', 'custom'}, optional
+
+            - '2d'
+                Uses :math:`2^d` different restarts, where :math:`d` is the
+                dimension of the datasets. In particular, tries all matrices
+                that are simultaneously diagonal and orthogonal. In other
+                words, these are diagonal matrices with all entries on the
+                diagonal being either +1 or -1. The final result is picked
+                based on the final values of the objective function. This is
+                motivated by the fact that in the graph setting, the embeddings
+                have two types of orthogonal non-identifiability, one of which
+                is captured by the orthogonal diagonal matrices. For more on
+                this, see [2]_.
+            - 'sign_flips'
+                Initial alignment done by making the median value in each
+                dimension have the same sign". The motivation is similar to
+                that in '2d', except this is a heuristic that can save time,
+                but can sometimes yield suboptimal results.
+            - 'custom'
+                Expects either an initial guess for :math:`Q` or an initial
+                guess for :math:`P`, but not both. See `initial_Q` and
+                initial_P`, respectively. If neither is provided, initializes
+                `initial_Q` to an identity with an appropriate number of
+                dimensions.
+
+        initial_Q : np.ndarray, shape (d, d) or None, optional (default=None)
+            An initial guess for the alignment matrix, Q, if such exists. Only
+            one of initial_Q, initial_P can be provided at the same time, and
+            only if `init` argument is set to 'custom'. If None, and initial_P
+            is also None - initializes `initial_Q` to identity matrix. Must be
+            an orthogonal matrix, if provided.
```

Comment:
thoughts on linking to the defn of an orthogonal matrix or describing it in a Notes section? @dwaynepryce is this clear as is?

Reply:
mentioned it in the Notes.

Reply:
but just a thought: if i stumbled upon some docs mentioning orthogonal matrices, and i had no idea what it is - my default would be to `Ctrl-t orthogonal matrix Return`, rather than `Ctrl-f orth Return`.

Reply:
fair enough, maybe unnecessary, i leave to @dwaynepryce for whatever will make sense to MSFT folks